astro/src/content/blog/announcing-fusionauth-1-49.mdx

```diff
@@ -0,0 +1,93 @@
+
+Therefore, I hereby dub 1.49 the "Bug fix Beluga" release.
+
+![Bug fixes are awesome.](/img/blogs/release-1-49/beluga-meme.png)
+
+All in all there are 27 issues, enhancements, and bug fixes included in the 1.49.0 release. As always, please see the [release notes](/docs/release-notes/#version-1-49-0) for a full breakdown of the changes between 1.48.3 and 1.49.0, including any schema changes.
+
+Let's look at some of the more notable fixes.
+
+## LinkedIn Provider Works Again
+
+In 2023, LinkedIn changed their implementation of ["Sign in with LinkedIn"](https://learn.microsoft.com/en-us/linkedin/consumer/integrations/self-serve/sign-in-with-linkedin-v2). They made it more OIDC compliant (yay!). They also kept the old version around for older apps (yay!). But it broke FusionAuth's [LinkedIn Identity Provider](/docs/lifecycle/authenticate-users/identity-providers/social/linkedin) for new LinkedIn integrations (boo!).
+
+The new version is what all new "Sign in with LinkedIn" integrations should use. The older version is [deprecated as of Aug 2023](https://learn.microsoft.com/en-us/linkedin/consumer/integrations/self-serve/sign-in-with-linkedin).
+
+In this release, FusionAuth uses the new LinkedIn Authentication API which is OIDC compatible. This includes more standard OIDC scopes, such as `openid` and `profile`, as well as a more standardized Id token which can be validated using [LinkedIn's discovery document](https://www.linkedin.com/oauth/.well-known/openid-configuration).
+
+We're happy LinkedIn updated their authentication workflow to support OIDC and are excited to support that integration going forward.
+
+## Elasticsearch Reindexing Progress
+
+I don't know about you, but I like progress indicators. I remember, years ago, watching progress indicators all the time, from network downloads to disk defragmentation programs. (RIP `Defrag`.) We all have faster internet and disk now, so hopefully those indicators are a thing of the distant past for you.
+
+But if you use FusionAuth and [reindex your Elasticsearch (or Opensearch) index](/docs/lifecycle/manage-users/search/search#reindexing-elasticsearch), before this release FusionAuth didn't do a great job of reporting progress. It only reported back every 250k objects, which depending on the data in the index, could give the impression that progress has halted.
+
+In this release and going forward, FusionAuth will report reindexing progress more regularly in order to spare everyone some frustration.
+
+## Email Link Workflow Improvements
+
+Speaking of frustration, FusionAuth has dealt with a lot of issues around [our magic links functionality](/docs/lifecycle/authenticate-users/passwordless/magic-links). This seems pretty straightforward:
+
+* create a customizable, localizable email template: check!
+* create a one-time use, random, high entropy string tied to a user account: check!
```

Comment:
Take or leave this one :D

```suggestion
* create a one-time use, random, high entropy string tied to a user account: ✅
```

Reply:
I'm going to pass on the emojis. I'm a curmudgeon, what can I say :).